lib/components/Information/Badge/index.tsx

```diff
@@ -1 +1,33 @@
-export * from "@/ui/badge"
+import { Avatar } from "@/components/Information/Avatar"
+import { cn } from "@/lib/utils"
+import { Badge as BadgeComponent } from "@/ui/badge"
+import { ComponentProps, forwardRef } from "react"
+
+interface BadgeProps {
+  text: string
+  hasAvatar?: boolean
+  variant?: ComponentProps<typeof BadgeComponent>["variant"]
+  imageSrc?: ComponentProps<typeof Avatar>["src"]
+  alt?: ComponentProps<typeof Avatar>["alt"]
+}
+
+export const Badge = forwardRef<HTMLDivElement, BadgeProps>(
+  ({ text, hasAvatar = false, imageSrc, alt, variant = "name" }) => {
```

Comment:
Can't we just use a property called `avatarSrc` and infer `hasAvatar` from there? Feels redundant.

Reply:
I had the same thought, but even if there's no avatarSrc, we could still have a fallback avatar that we want to display. So, you can provide no avatar and not alt text, but still want to show an image.

Reply:
I'll try it out